pkg/master/master.go

```diff
@@ -122,6 +123,8 @@ type Config struct {
 	DeleteCollectionWorkers int
 	EventTTL                time.Duration
 	KubeletClient           kubeletclient.KubeletClient
+	RESTStorageProviders    map[string]RESTStorageProvider
```

Comment:
> What is the key of this map? please document. Does this map really need a key?

`GroupName`.  Having it keyed ensures no collisions and makes it easy to later add "which groups are here".